proto/redpanda/api/dataplane/v1alpha1/secret.proto

```diff
@@ -207,8 +214,21 @@ message CreateConnectSecretRequest {
     (buf.validate.field).string.pattern = "^[a-zA-Z0-9-_]+$"
   ];
 
-  // The input for the secret to create.
-  SecretInput secret = 3 [(google.api.field_behavior) = REQUIRED];
+  // Secret labels.
+  map<string, string> labels = 3 [
+    (buf.validate.field).map = {
+      values: {
+        string: {pattern: "^([\\p{L}\\p{Z}\\p{N}_.:/=+\\-@]*)$"}
+      },
+    },
+    (google.api.field_behavior) = IMMUTABLE
+  ];
+
+  // The secret data.
+  bytes secret_data = 4 [
```

Comment:
Should we add validation for the max length of the secret data (seems 64KiB is enforced on AWS and GCP) for update and create?

Reply:
Added.